packages/db/src/types.ts

```diff
@@ -81,7 +83,7 @@ export interface PostgresProvider {
 export interface PostgresProviderRecord extends PostgresProvider {
   created_at: number,
   updated_at: number,
-  cumulusId: number,
+  cumulus_id: number
```

Comment:
This is a nit too - any reason why you removed the comma? 
```suggestion
  cumulus_id: number,
```
I don't mind either way but I'd like to be consistent.